src/components/FetchHandler.tsx

```diff
@@ -27,28 +25,27 @@ export const FetchHandler = <TResult extends {}>({
     notFoundIndicator,
     loadingIndicator,
 }: Props<TResult>) => {
-    const { loading, data, error, trigger, errorCode } = usePromise(apiMethod);
-    const [dependencyList, setDependencyList] = useState(undefined);
+    const { loading, data, error, trigger } = usePromise(apiMethod);
 
     useEffect(() => {
         trigger();
-    }, [dependencyList, trigger]);
+    }, [trigger]);
 
     const renderError = () => {
-        if (errorIndicator && (errorCode !== 404 && notFoundIndicator)) {
+        if (errorIndicator && error?.code !== 404 && notFoundIndicator) {
             return errorIndicator;
-        } else if (errorCode === 404 && notFoundIndicator) {
+        } else if (error?.code === 404 && notFoundIndicator) {
             return notFoundIndicator;
         }
 
-        return errorCode !== 401 ? <Alert type="error" message={errorText} /> : <Spinner centered />;
+        return error?.code !== 401 ? <Alert type="error" message={errorText} /> : <Spinner centered />;
     };
```

Comment:
Varför måste det vara en Alert här? Känns som vi borde få välja vad för komponent som ska visas i dessa scenarios. notFoundIndicator exempelvis en valbar prop som vi kan välja att ge en valfri komponent till. men här nere kan vi inte det?

Reply:
Translate : "Why does it have to be an Alert here? Feels like we should choose what component to display in these scenarios. notFoundIndicator is an example of a selectable prop that we can choose to provide an optional component to. but down here we can't?"

Reply:
It's out of scope for this fix.